00-RELEASENOTES

```diff
@@ -1,16 +1,284 @@
+
+New/Modified Metrics - Core
+===========================
+* Introduce per-client metrics for network traffic and command execution
+  in `CLIENT LIST` and `CLIENT INFO`. (#327)
+* Add metrics for DB memory overhead and rehashing status to `INFO MEMORY`
+  and `MEMORY STATS`. (Redis#12913)
+* Add `pubsub_clients` metric to `INFO CLIENTS`. (Redis#12849)
+* Add metrics for client buffer limit disconnections to `INFO`. (Redis#12476)
+* Add metrics for monitoring clients using `WATCH` command and watched keys.
+  (Redis#12966)
+
+New Features - Core
+===================
+* Support replica redirect for read/write operations to primary in standalone
+  mode. (#325)
+* Add server config for cluster blacklist TTL. (#738)
+* Add availability zone server config. (#700)
+
+New Features - Cluster
+======================
+* Support IPv4 and IPv6 dual stack and client-specific IPs in clusters. (#736)
+* Support `BY/GET` options for `SORT/SORT_RO` in cluster mode when pattern
+  implies a single slot. (Redis#12728)
+
+Compatibility Improvements - Core
+=================================
+* Derive RDB and module child process names based on server start name for
+  compatibility. (#454)
+* Update server identity in `serverPanic` output based on `extended-redis-compatibility`
+  config. (#415)
+
+Compatibility Improvements - Sentinel
+=====================================
+* Accept `redis-sentinel` to start Valkey in sentinel mode. (#731)
+
+Performance/Efficiency Improvements - Core
+==========================================
+* Introduce dual channel for more efficient full sync replication. (#60)
+* Introduce async IO threading for improved multi-threaded performance.
+  (#763, #758)
+* Embed key directly in main dictionary entry for improved memory efficiency.
+  (#541)
+* Use thread-local storage to reduce atomic contention in updating memory
+  metrics. (#674)
+* Reduce redundant calls to `prepareClientToWrite` for continuous `addReply*`.
+  (#670)
+* Optimize the logic for checking conversion to skip list during `ZADD` operations.
+  (#806)
+* Optimize `sdsfree` with `zfree_with_size` to avoid redundant size calculation.
+  (#453)
+* Combine events to eliminate redundant `kevent(2)` calls. (#638)
+* Introduce shared query buffer for client reads to reduce memory usage. (#258)
+* Optimize CRC64 performance for large batches by processing bytes in parallel.
+  (#350)
+* Use `SOCK_NONBLOCK` to reduce system calls for outgoing connections. (#293)
+* Enable `accept4()` detection on specific versions of various platforms. (#294)
+* Convert CRC16 slot table to fixed-size array for improved memory efficiency.
+  (Redis#13112)
+* Run `SCRIPT FLUSH` truly asynchronously and close Lua interpreter in a
+  background thread. (Redis#13087)
+* Optimize `DEL` command to avoid redundant deletions for expired keys. (Redis#13080)
+* Improve defragmentation for large bins to enhance memory efficiency. (Redis#12996)
```

Comment:
This also introduced a new `allocator_muzzy`.

Reply:
added to the "New/Modified Metrics - Core" section